src/GrpcProtos/capture.proto

```diff
@@ -64,6 +64,7 @@ message CaptureOptions {
   DynamicInstrumentationMethod dynamic_instrumentation_method = 18;
 
   repeated InstrumentedFunction instrumented_functions = 5;
+  map<uint64, uint64> absolute_address_to_size_of_functions_to_stop_at = 19;
```

Comment:
Maybe this is not really a problem, but I don't really like maps in protos. Also we have the change of making the type more structured. I think I would prefer the following:
- Define a message `FunctionToStopUnwindingAt`.
- Make this a `repeated FunctionToStopUnwindingAt` field.

The translation to a map can still be done in `TracerImpl::TracerImpl`.

Reply:
As discussed offline, done.